content/de/docs/tasks/tools/included/kubectl-whats-next.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Wie geht es weiter?"
+description: "Wie geht es nach der Installation von Kubectl weiter."
+headless: true
+_build:
+  list: never
+  render: never
+  publishResources: false
+---
+
+* [Minikube installieren](https://minikube.sigs.k8s.io/docs/start/)
+* [Installations Guides](/docs/setup/) ansehen um mehr über die Clustererstellung zu erfahren.
+* [Lernen wie man Applikationen startet und erreichbar macht.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
+* Falls Sie Zugriff auf ein Cluster benötigen, welches Sie nicht selber erstellt haben, könnte 
```

Comment:
We decided to use an informal tone to address the readers "Du" vs "Sie" -> https://kubernetes.io/de/docs/contribute/localization/#informale-schreibweise

Reply:
haven't read this guideline. Sorry. I saw some other translations with "Sie" so i thougt it could be right. I will change it soon.